app/facade/food_facade.rb

```diff
@@ -0,0 +1,13 @@
+class FoodFacade
+  def self.search(food)
+    json = FoodService.search_food(food)
+    json[:foods].map do |food_data| 
```

Comment:
This looks good. Could we refactor this to single method that gives us access to all of the info we need/want?